src/sidebar-container.component.ts

```diff
@@ -260,7 +248,7 @@ export class SidebarContainer implements AfterContentInit, OnChanges, OnDestroy
     if (this._sidebars && this.allowSidebarBackdropControl) {
       let hasOpen = false;
 
-      const _sidebars = this._sidebars.toArray();
+      const _sidebars: Sidebar[] = [...this._sidebars];
```

Comment:
Posibibly not. We can use directly 
`
const _sidebars = this._sidebars;
`

or use the property of the class directly as for argument.